### ISSUE TITLE
Update screenshots

### ISSUE THREAD

**Issue by valoeghese** [SuperCoder7979/simplexterrain#26]
Version 0.1.0 screenshots bad and worst in game

**Comment by yavincl** [comment]
I took some while playing around with Worley's noise algorithm. Feel free to use them.
![2020-07-17_19 39 04](https://user-images.githubusercontent.com/57966308/87859493-f5ea4c80-c924-11ea-92f0-5b8b703504ea.png)
![2020-07-18_01 17 03](https://user-images.githubusercontent.com/57966308/87859496-f84ca680-c924-11ea-87cb-5f26438d5090.png)
![2020-07-17_17 30 59](https://user-images.githubusercontent.com/57966308/87859501-fedb1e00-c924-11ea-9b98-6917079786f6.png)

**Comment by jaskarth** [comment]
the middle one looks good!

**Comment by yavincl** [comment]
Yep, it turns out you can still have beautiful beaches and large seas with Worley if the difference between main and low amplitudes is low and at least Low is negative. Like:
```
  "mainAmplitudeHigh": -69.0,
  "mainAmplitudeLow": -245.0,
```
Or
```
  "mainAmplitudeHigh": 42.0,
  "mainAmplitudeLow": -69.0,
```
